### PR TITLE
Create hl-inside.ru.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

**Comment by J0WI** [comment]
The site has some mixed content issues.

**Comment by koops76** [comment]
@J0WI Which domains should I exclude?

**Comment by J0WI** [comment]
`bbs`, `forum` and `forums` are affected.

**Comment by koops76** [comment]
@J0WI Do not merge yet, will add a `platform="mixedcontent"` ruleset.

File: src/chrome/content/rules/hl-inside.ru-mixedcontent.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ruleset name="hl-inside.ru (mixed content)" platform="mixedcontent">
```
Comment (J0WI) [inline src/chrome/content/rules/hl-inside.ru-mixedcontent.xml]: Please link this ruleset with hl-inside.ru.xml trough a comment.

Comment (koops76) [inline src/chrome/content/rules/hl-inside.ru-mixedcontent.xml]: @J0WI Done.

**Pull request merged**